src/Lib.P2P.Tests/Transports/TcpTest.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Lib.P2P.Transports;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MultiFormats;
+
+namespace Lib.P2P.Tests.Transports
+{
+    [TestClass]
+    public class TcpTest
+    {
+        [TestMethod]
+        public void Connect_Unknown_Port()
+        {
+            var tcp = new Tcp();
+            ExceptionAssert.Throws<SocketException>(() =>
+            {
+                var _ = tcp.ConnectAsync("/ip4/127.0.0.1/tcp/32700").Result;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)